openvdb_houdini/houdini/SOP_OpenVDB_Sample_Points.cc

```diff
@@ -265,6 +302,24 @@ newSopOperator(OP_OperatorTable* table)
             "A subset of the input VDBs to be processed"
             " (see [specifying volumes|/model/volumes#group])"));
 
+    // vdb points groups
+    parms.add(hutil::ParmFactory(PRM_STRING, "vdbpointsgroups", "VDB Points Groups")
+        .setHelpText("Specify subset of VDB points to sample."
+                     " This parameter has no effect if there are no input VDB point grids")
+        .setChoiceList(&hvdb::VDBPointsGroupMenuInput1));
+
+    // rename "vel" attribute to "v"
+    parms.add(hutil::ParmFactory(PRM_TOGGLE, "renamevel", "Rename Vel To V")
```

Comment:
It would probably be a fairly minor change to support this for Houdini points as well? I think for consistency that would be good, because the comment about it being VDB Points only in the help text could be easily missed, causing some confusion. Admittedly that could break existing scenes, so perhaps making this default to off would be preferred to having functionality apply just to VDB points.